saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/registration/OpenSamlRelyingPartyRegistration.java

```diff
@@ -48,7 +48,8 @@ public final class OpenSamlRelyingPartyRegistration extends RelyingPartyRegistra
 		super(registration.getRegistrationId(), registration.getEntityId(),
```

Comment:
Will you please remove this class in this PR as well? Please place it in a separate commit so that it can easily be restored in the event that a concern arises.